### DISCUSSION TITLE
Interactive mode

### DISCUSSION THREAD

**Discussion by TonyHacker03** [williamcorsel/hassio-addons#13] [Q&A] [answered]
is there any method to start the docker image in interactive mode directly from addon, or any other method to have access to minecraft terminal? (bedrock server)

**Comment by williamcorsel** [comment, marked as answer]
This is not possible in HA in general. You could check the Portainer add-on: https://github.com/alexbelgium/hassio-addons/tree/master/portainer which does allow you to see and access all running containers.

**Reply by TonyHacker03** [reply]
Yes, I already installed portainer on my ha but when i try to do the "attach" command it says "the image was not started with interactive flag"

**Reply by TonyHacker03** [reply]
My question is: is there any way to start the docker container in interactive mode from home assistant and access to Minecraft terminal from portainer?
Sorry for the bad english i'm italian
